packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -20,8 +23,40 @@ export interface BoxProps
     PositionProps,
     FlexboxProps {
   as?: keyof JSX.IntrinsicElements;
+  content?: string;
+  cursor?: CursorProperty;
```

Comment:
I don't know how content is working, but for cursor and appearance you have compose a couple of `system` styles into the component for them to work by the looks of it. Similar to what Chakra are doing with this object https://github.com/chakra-ui/chakra-ui/blob/master/packages/chakra-ui/src/Box/config.js#L113

I created a little cut down object locally and it seemed to get it working as expected.

```
const config = {
  textTransform: true,
  animation: true,
  appearance: true,
  transform: true,
  transformOrigin: true,
  visibility: true,
  whiteSpace: true,
  userSelect: true,
  pointerEvents: true,
  wordBreak: true,
  overflowWrap: true,
  textOverflow: true,
  boxSizing: true,
  cursor: true,
  resize: true,
  transition: true,
  listStyleType: true,
  listStylePosition: true,
  listStyleImage: true,
  objectFit: true,
  objectPosition: true,
  outline: true,
  float: true,
  willChange: true,
};

const extraConfig = system(config);
```

then add to the compose

```
export const Box = styled.div(
  {
    boxSizing: 'border-box',
    minWidth: 0,
  },
  compose(
    space,
    layout,
    flexbox,
    background,
    backgroundColor,
    border,
    borderColor,
    boxShadow,
    position
  ),
  getPseudoStyles
) as React.FC<BoxProps>;
```

Reply:
Added a bunch. Good catch